build/lib/release.sh

```diff
@@ -187,7 +187,9 @@ function kube::release::package_node_tarballs() {
     cp "${client_bins[@]/#/${LOCAL_OUTPUT_BINPATH}/${platform}/}" \
       "${release_stage}/node/bin/"
 
-    cp "${KUBE_ROOT}/Godeps/LICENSES" "${release_stage}/"
+    cp -R "${KUBE_ROOT}/vendor/ALL_LICENSES" "${release_stage}/"
+    cp "${KUBE_ROOT}/LICENSE" "${release_stage}/"
```

Comment:
why are we copying this file to 2 different names?  At the very least this needs a comment.

Reply:
There are severals cases which depends on this file ${release_stage}/LICENSES, such as 
"pull-kubernetes-e2e-gce-100-performance
pull-kubernetes-e2e-gce-device-plugin-gpu"

As dims' suggestion, I just copied the LICENSES file to there.